src/compiler/sys/in-memory-fs.ts

```diff
@@ -221,7 +221,7 @@ export const createInMemoryFs = (sys: d.CompilerSystem) => {
   const emptyDirs = async (dirs: string[]): Promise<void> => {
     dirs = dirs
       .filter(isString)
-      .map(normalizePath)
+      .map((s) => normalizePath(s))
```

Comment:
EDIT: NVM - I found the change later in the review to `normalizePath`, and GH didn't delete it (or that's my story at least 😏 )

Reply:
yeah I wish this could stay as a unary function here, but I think the issue here is that `.map` will pass more than one argument and the type of the second argument won't match the second argument of `normalizePath` anymore, so arrow function it is